src/block/separator/style.scss

```diff
@@ -5,6 +5,21 @@
  */
 @import "common";
 
+@media (max-width: 1024px){
+  	
+	.ugb-separator .ugb-block-content .ugb-separator__svg-wrapper {
+	height : 175px;
+}
+	
+  }
+  @media (max-width: 767px){
+  
+
+	.ugb-separator .ugb-block-content .ugb-separator__svg-wrapper {
+		height:75px;
+	}
+	
+  }  
```

Comment:
Please clean this up & follow convention in other css files.  No blank lines, spaces between selector & curly brackets